main.cpp

```diff
@@ -465,33 +487,39 @@ int main()
     crushIt.setPersonalTrainer("chuck", 2.5f, 1000);
     Gym::Member example;
     crushIt.sellMonthlyPass(example, 29.99, 6);
-    std::cout << "average user number is: " << crushIt.getUserFreq(45, 30) << "\n";
+    std::cout << "average user number is: " << crushIt.getUserFreq(45, 30) << ", today there were: " << crushIt.mNumDailyPeople << "\n";
+    crushIt.printUserFreq();
     crushIt.checkIncome(34, 30, 24.99);
     
     Gym::Member patron;
+    std::cout << "you have " << patron.passesLeft << " passes left!\n";
     patron.punchPass(5);
     patron.bringAFriend(true, 4);
     patron.giveReward(1, "10% Discount", 10);
     patron.trackProgress(2, 10);
     
     Piano grandPiano;
     grandPiano.volumeDown(false, 1);
```

Comment:
`volumeDown` is not a verb. 
choose function names that describe actions. 
`decreaseVolume` describes an action, as does `adjustVolume` or `changeVolume`
- [ ] revise